test-app-util.gradle

```
@@ -21,8 +21,8 @@ import java.nio.file.Paths
 * The search begins at the given base directory (a File object). The returned
 * path is a string.
 */
-ext.findNodeModulePath = { packageName ->
-    def basePath = rootDir.toPath().normalize()
+ext.findNodeModulePath = { baseDir, packageName ->
```

`findNodeModulesPath`?
```suggestion
ext.findNodeModulesPath = { baseDir, packageName ->
```